### PR TITLE
rename MapeoManager.start() and MapeoManager.stop() methods

### PR DESCRIPTION
**Pull request by achou11**
Closes https://github.com/digidem/mapeo-core-next/issues/391

Renames them to `startMediaServer()` and `stopMediaServer()`, respectively

### CODE REVIEW

File: test-e2e/media-server.js [hunk 1]
```diff
@@ -19,6 +20,27 @@ const projectMigrationsFolder = new URL('../drizzle/project', import.meta.url)
...
+test('start/stop lifecycle', async (t) => {
+  const manager = new MapeoManager({
+    rootKey: KeyManager.generateRootKey(),
+    projectMigrationsFolder,
+    clientMigrationsFolder,
+    dbFolder: ':memory:',
+    coreStorage: () => new RAM(),
+  })
+
+  await manager.startMediaServer()
```
Comment (gmaclennan) [inline test-e2e/media-server.js:32]: Check it's started: get a url, check it works.

File: test-e2e/media-server.js [hunk 2]
```diff
@@ -19,6 +20,27 @@ const projectMigrationsFolder = new URL('../drizzle/project', import.meta.url)
...
+  await manager.startMediaServer()
```
Comment (gmaclennan) [inline test-e2e/media-server.js:33]: get a url, check the port hasn't changed

File: test-e2e/media-server.js [hunk 3]
```diff
@@ -19,6 +20,27 @@ const projectMigrationsFolder = new URL('../drizzle/project', import.meta.url)
...
+  await manager.stopMediaServer()
```
Comment (gmaclennan) [inline test-e2e/media-server.js:34]: check the url previously returned no longer works (should return a connection error)

File: test-e2e/media-server.js [hunk 4]
```diff
@@ -19,6 +20,27 @@ const projectMigrationsFolder = new URL('../drizzle/project', import.meta.url)
...
+
+  await manager.startMediaServer()
+  await manager.stopMediaServer()
```
Comment (gmaclennan) [inline test-e2e/media-server.js:37]: Checks as above

File: test-e2e/media-server.js (hunk 1 above)
Comment (gmaclennan) [reply test-e2e/media-server.js:32]: Don't need to put data there, just returning 404 is enough to know it's started.

**Review by gmaclennan — APPROVED** [review]
Looks good, although now you've added the lifecycle test I want to actually make it test the server is going through the correct lifecycle. non-blocking though I would say, could be done later.

**Comment by achou11** [comment]
> Looks good, although now you've added the lifecycle test I want to actually make it test the server is going through the correct lifecycle. non-blocking though I would say, could be done later.

addresse in https://github.com/digidem/mapeo-core-next/pull/399/commits/876b628762cac6a048432325ad8de53457a8554d

**Pull request merged**